Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
 name = "tomlq"
-version = "0.1.6"
+version = "0.2.0"
 authors = [
     "Natalia Maximo <iam@natalia.dev>",
     "James Munns <james.munns@gmail.com>",
+    "Venus Xeon-Blonde <venus@venusblon.de>"
```

Comment:
I've usually seen the Authors field used to denote original authors + current maintainers. I'm open to chatting about having additional maintainers, especially since it looks like you have at least a few ideas, but I would rather wait til we've chatted before listing someone as an author on their first contribution.